java/client/src/org/openqa/selenium/print/BUILD.bazel

```diff
@@ -0,0 +1,12 @@
+load("//java:defs.bzl", "java_library")
+
+java_library(
+    name = "print",
+    srcs = glob(["**/*.java"]),
```

Comment:
`*.java`. Globbing to an indeterminate depth is seldom what we want to do.

Reply:
This file isn't needed at all